spring-data-relational/src/main/java/org/springframework/data/relational/core/mapping/AggregatePath.java

```diff
@@ -338,4 +429,102 @@ static ColumnInfo of(AggregatePath path) {
 			return new ColumnInfo(columnName, AggregatePathTableUtils.prefixWithTableAlias(path, columnName));
 		}
 	}
+
+	/**
+	 * A group of {@link ColumnInfo} values referenced by there respective {@link AggregatePath}. This is relevant for
+	 * composite ids and references to such ids.
+	 **/
```

Comment:
`**/`?
Also, missing `@since` and method documentation.

Reply:
Fixed